From efefcbe9d0e8b356af822200189c6a3b25300505 Mon Sep 17 00:00:00 2001
From: Vladimir Oltean <olteanv@gmail.com>
Date: Wed, 26 Sep 2018 23:33:32 +0300
Subject: [PATCH 03/17] common: grant cnss-daemon access to sysfs_net

* It's scanning the /sys/class/net directory tree
* Yes, it's actually labeled as wcnss_service.

Change-Id: I2227868405d9ea6bfba1b9fa6103225b134c5759
Signed-off-by: Vladimir Oltean <olteanv@gmail.com>
---
 common/wcnss_service.te | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/wcnss_service.te b/common/wcnss_service.te
index 5e9d52c..a8f957a 100644
--- a/common/wcnss_service.te
+++ b/common/wcnss_service.te
@@ -43,6 +43,7 @@ allow wcnss_service firmware_file:file r_file_perms;
 allow wcnss_service sysfs:file rw_file_perms;
 allow wcnss_service sysfs_data:file r_file_perms;
 allow wcnss_service proc:file r_file_perms;
+r_dir_file(wcnss_service, sysfs_net);
 
 # allow access to network performance tuner
 unix_socket_connect(wcnss_service, wigignpt, wigignpt)
-- 
2.17.1

